content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+---
+date: "2020-02-20T00:00:00Z"
+lastmod: "2020-02-20T00:00:00Z"
+title: "Deploy an HA Cluster"
+weight: "1005"
+---
+
+In this guide, we'll walk through the steps needed to take enable Kubernetes' high availability capabilities with a cluster installed by kURL and managed by KOTS. 
+This only applies to 'embedded' cluster installs using Kurl.
+
+## Objective
+
+As with the rest of the guides, this is meant as a 'hello world' example to help get you familiarized with the process of standing up a highly available Kubernetes cluster using the Replicated toolset. The architecture we will follow is presented below:
+
+![Architecture](/images/guides/kots/ha-cluster-architecture.png)
+
+The diagram shows 4 Virtual Machines (VMs) running on Google Cloud Platform (GCP), with one of them used solely as a load balancer. Note that there aren't any requirements to use GCP. You can follow along with VMs provisioned on a different cloud provider or on premise.
+
+## Overview
+
+The guide is broken into two parts. The [first part](#part-i---setting-up-the-cluster) will walk you through how to stand up the cluster, while the [second part](#part-ii-testing-scenarions--troubleshooting) will go through a couple of testing scenarios to validate the cluster's resiliency and some troubleshooting tips.
+
+There are a few things to keep in mind about this guide:
+
+- All VMs are going to be created in Google Cloud Platform (GCP) but nothing in this guide is dependent on GCP specific services. 
+- The examples in this guide used ephemeral resources.
```

Comment:
What exactly do you mean by ephemeral resources? Servers? IPs? everything?

Reply:
No I am thinking there is really no point to even mention this... so I have removed that bullet point.